src/main/java/org/opentripplanner/routing/algorithm/raptoradapter/transit/mappers/TripPatternForDateMapper.java

```diff
@@ -53,6 +53,18 @@ public class TripPatternForDateMapper {
   public TripPatternForDate map(Timetable timetable, LocalDate serviceDate) {
```

Comment:
Add the @Nullable annotation to the method to indicated this fact to the callers.